medusa/indexers/tvdbv2/tvdbv2_api.py

```diff
@@ -629,20 +611,19 @@ def get_last_updated_seasons(self, show_list, from_time, weeks=1):
             total_updates = []
             # Get the shows episodes using the GET /series/{id}/episodes route, and use the lastUpdated attribute
             # to check if the episodes season should be updated.
-            logger.debug('Getting episodes for {show_id}', show_id=show_id)
+            log.debug('Getting episodes for {0}', show_id)
             episodes = self._download_episodes(show_id)
 
             for episode in episodes['episode']:
-                if episode.get('seasonnumber') is None or episode.get('episodenumber') is None:
-                    logger.warning('This episode has incomplete information. The season or episode number '
-                                   '(season: %s, episode: %s) is missing. '
-                                   'to get rid of this warning, you will have to contact tvdb through their forums '
-                                   'and have them fix the specific episode.',
-                                   episode.get('seasonnumber'), episode.get('episodenumber'))
+                seasnum = episode.get('seasonnumber')
+                epno = episode.get('episodenumber')
+                if seasnum is None or epno is None:
+                    log.warning('Invalid episode numbering (season: {0!r}, episode: {1!r}) '
```

Comment:
repr is not strictly necessary for these since we are looking for None specifically, however my personal preference for logging an invalid value of any sort is repr since, for example, '0' and 0 are not the same thing